resources/assets/coffee/react/_components/user-badge.coffee

```diff
@@ -19,8 +19,19 @@
 import * as React from 'react'
 import { div } from 'react-dom-factories'
 bn = 'user-group-badge'
+group_names =
+  bot: 'Automated Message'
```

Comment:
I'm pretty sure that "automated message" won't really work globally, would be better to replace it with something more universal so it doesn't fit on modding discussions (and possibly some other message stuff) only.

Reply:
also pretty sure this one will want to be localisable

Reply:
so finally should those strings be localisable or not?